### DISCUSSION TITLE
2024 Q1 POAP Eligibility List and Contributor Incentives

### DISCUSSION THREAD

**Discussion by darwintree** [Conflux-Chain/conflux-documentation#514] [Announcements]
## POAP Eligibility List POAP 领取资格名单

We are excited to announce the eligibility list of contributors who are qualified to claim Proof of Attendance Protocol (POAP) tokens for 2024 Q1. We deeply appreciate your support and highly value the insights and expertise you bring to our community.
以下为2024 Q1季度有资格领取 POAP 的贡献者名单。非常感谢名单中的各位对本项目的支持。

The following Crowdin contributors are eligible:
以下的Crowdin贡献者有资格领取POAP：

- [effectchen](https://crowdin.com/profile/effectchen)
- [louisl.eth](https://crowdin.com/profile/louisl.eth)
- [xbl.zz.sxm](https://crowdin.com/profile/xbl.zz.sxm)
- [sys555](https://crowdin.com/profile/sys555)
- [conflux.pei](https://crowdin.com/profile/conflux.pei)
- [Yumingyuan](https://crowdin.com/profile/Yumingyuan)
- [w33d](https://crowdin.com/profile/w33d) (distinguished)
- [devBlue4](https://crowdin.com/profile/devBlue4) (distinguished)
- [lllu23](https://crowdin.com/profile/lllu23) (distinguished)
- [PingJing](https://crowdin.com/profile/PingJing) (distinguished)
- [jasonyou184](https://crowdin.com/profile/jasonyou184) (distinguished)
- [Axxx036](https://crowdin.com/profile/Axxx036) (distinguished)

The following GitHub contributors are eligible:
以下的GitHub贡献者有资格领取POAP：

- [Pana](https://github.com/Pana)
- [darwintree](https://github.com/darwintree)
- [0xn1c0](https://github.com/0xn1c0)
- [confluxdevops](https://github.com/confluxdevops)
- [Last-emo-boy](https://github.com/Last-emo-boy)
- [ClockRide](https://github.com/ClockRide)
- [TylerSehr](https://github.com/TylerSehr)
- [devblue4](https://github.com/devblue4)
- [intrepidcanadian](https://github.com/intrepidcanadian) (distinguished)
- [jackleeio](https://github.com/jackleeio) (distinguished)

The eligibility list is curated in accordance with our [CONTRIBUTING guideline](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING). If you believe your contribution has been overlooked, please do not hesitate to contact us.
白名单是根据[CONTRIBUTING指导原则](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING)所挑选的。如果您有对应贡献但被遗漏在了名单外，欢迎联系我们。

The mint deadline is block number 230854533 (May 9th 2024 09:38:00 UTC+08:00).
铸造的截止时限为区块号 230854533（2024年05月09日 09:38:00 UTC+08:00）。

<details>
<summary>For Newcomers: How to Qualify for Minting the POAP </summary>
To qualify for minting the POAP, your contribution must have been "ACCEPTED", which means:

* Any issue has been created with the "ACCEPTED" label added by an admin
* Any Pull Request has been successfully merged into the main branch
* Any translation string has been thoroughly reviewed

</details>
<details>
<summary>对新参与者的提示：如何有资格铸造POAP</summary>
要想有资格铸造POAP，你的贡献必须被"接受"，即：

* 管理员添加了"ACCEPTED"标签的任何issue
* 成功合并到主分支的任何Pull Request
* 被彻底审查的任何翻译字符串（Crowdin）

</details>

Please be aware that the POAP issuance rules are subject to review and potential modifications every 3 months. This ensures that our guidelines remain pertinent and effective in encouraging and rewarding valuable contributions.
请注意，POAP的发行规则每3个月会进行审查和可能的修改，以确保我们的指导规则仍然可以有效地鼓励和奖励社区的贡献。

## Contributor Incentives 贡献者激励

In addition to POAP, this quarter, we have introduced further financial incentives for contributors to our repository, this includes both translation contributors and documentation content contributors. These incentives are **directly** related to the quality of the contributor's work.

除了POAP外，本季度我们还为仓库的贡献者（包括翻译贡献者与文档内容贡献者）引入了更多经济激励。这些激励是与贡献者的工作质量所**直接**相关的。

At present, we offer corresponding incentives for Chinese translators. Translator incentives are calculated based on the volume of translations and whether the translation is accepted. We will airdrop rewards to translators who have provided their addresses. For the method of providing addresses, please refer to [Conflux Technical Documentation Translation Plan (2024.03.06) ](https://forum.conflux.fun/t/conflux-2024-03-06/20286). It should be noted that this airdrop will also provide incentives for translators before January 2024. It is expected that the translator incentives will be fully distributed before April 20th.

目前我们为中文翻译者提供相应激励，翻译者激励计算与翻译量与翻译是否被接受相关。我们将为已提供了地址的翻译者空投激励。地址提供方式请参考[Conflux 技术文档翻译计划（2024.03.06）](https://forum.conflux.fun/t/conflux-2024-03-06/20286)。需要说明的是，本次空投也会为2024年1月前的翻译者提供激励。预计本轮翻译者激励将在4月20日前发放完毕。

For incentives for documentation content contributors, please refer to [Conflux Doc Contributions DB (Jan 2024 - Mar 2024)](https://confluxglobal.notion.site/8e3343b46c214221bf620f46b2f61795?v=111e1e5e40124195b1c8898ef1a56bd9&pvs=4), and their incentives are calculated according to the [2024Q1 reward guidelines](https://github.com/Conflux-Chain/conflux-documentation/blob/2024Q1-reward/docs/general/CONTRIBUTING/reward-guidelines.md). If you have relevant contributions but have been omitted from the list, please feel free to contact us. In addition, we will collect the (temporary) addresses provided by the relevant contributors for airdrop redemption in this discussion.

文档内容贡献者的激励请参考[Conflux Doc Contributions DB (Jan 2024 - Mar 2024)](https://confluxglobal.notion.site/8e3343b46c214221bf620f46b2f61795?v=111e1e5e40124195b1c8898ef1a56bd9&pvs=4)，其激励根据[2024Q1 reward guidelines](https://github.com/Conflux-Chain/conflux-documentation/blob/2024Q1-reward/docs/general/CONTRIBUTING/reward-guidelines.md)计算。如果您有对应贡献但被遗漏在了名单外，欢迎联系我们。此外我们将在本discussion中收集相关贡献者提供的领取空投的（临时）地址。

> Please note that specific details will be adjusted in the new quarter, for details, please refer to [regular review of reward guideline](https://github.com/Conflux-Chain/conflux-documentation/pull/508).
>
> 需要说明的是，新的季度中，特定细则将会调整，详情请参考[regular review of reward guideline](https://github.com/Conflux-Chain/conflux-documentation/pull/508)。

Thank you for your assistance in enhancing our project. If you have any queries or require assistance, please feel free to create an issue in our Github repository.
感谢各位持续帮助我们改进项目。如果你还有任何疑问或需要帮助，请随时在我们的Github仓库中创建issue。

### How to Claim the Contributor POAP 如何领取贡献者 POAP

1. Visit the [Conflux Document POAP Portal](https://dual-space-nft-frontend.vercel.app). 访问 [Conflux Document POAP Portal](https://dual-space-nft-frontend.vercel.app)。
2. Choose the space(s) in which you want to mint the POAP. You can choose Core Space, eSpace, or both (Dual Space). It is recommended to select core space or dual space because FC reward is only claimable on Conflux Core Space. 选择你想铸造到的 Space，你可以铸造到Core Space，eSpace 或者是同时铸造到两个 Space。我们推荐你铸造到 Core Space 或同时铸造到两个 Space 因为在 Core 空间能够领取 NFT 相关联的 FC 奖励。
3. Input your wallet address OR connect your wallet to automatically fill in the address. 输入你的钱包地址或者连接你的钱包以自动输入。
4. Choose the appropriate authorization method (GitHub or Crowdin). 选择合适的授权方式（Github 或 Crowdin）。
5. Mint your POAP! 现在即可铸造你的POAP。

<img width="1270" alt="image" src="https://github.com/Conflux-Chain/conflux-documentation/assets/17946284/e649ca37-1acc-47c1-894c-760456110e96">

> Contact us if you are notified to have no permission.

> 如果你被提示无权限进行铸造，请联系我们。

### How to Get POAP-Attached FC Rewards 如何领取 POAP 关联的 FC 奖励

We have set up a [Contributor Rewards Pool](https://confluxscan.io/address/cfx:achm6kb92by13rpvwd04zscn1127zuaseu25usm7sc) to reward holders of Proof of Attendance Protocols (POAPs) with FansCoin. This is the POAP-attached incentive and is calculated independently from other incentives.

我们设立了一个[贡献者奖励池](https://confluxscan.io/address/cfx:achm6kb92by13rpvwd04zscn1127zuaseu25usm7sc)，以用FansCoin奖励文档POAP的持有者。这一激励为POAP持有者提供附加的奖励，与其他激励独立结算。

1. Make sure your core account owning POAP is connected to the [Conflux Document POAP Portal](https://dual-space-nft-frontend.vercel.app). Your account is supposed to own CFX to pay for gas and storage collateral. 确认你已经将 Core 空间账户连接到了 [Conflux Document POAP Portal](https://dual-space-nft-frontend.vercel.app)。你的账户上需要有一定的CFX用于支付gas与存储抵押。
2. Click "Core Manage" on the left, you will see your POAP listed. 点击左侧的 "Core Manage"，你会看到 Core 账户持有的 POAP。
3. Select "Token ID" in the "Claim FC Reward" card, it will show how much you could claim. 在"Claim FC Reward"卡片中选择"Token ID"，将会显示你可以领取多少FC。
4. Click the "Claim xx FC" button and your wallet will be invoked. You can claim FC for each POAP you own (the FC amount available for claim may vary).点击"Claim xx FC"按钮，你的钱包将会被唤起。对于你持有的每个POAP，你都可以领取一次（不同的POAP可领取的数量会有所区别）。

<img width="1285" alt="image" src="https://github.com/Conflux-Chain/conflux-documentation/assets/17946284/d73f17b4-76e7-44b0-9b7c-f47edd88a45e">

> For the POAP issued in the previous quarters, you can visit "Set Core Owner" card in "eSpace Manage" to set the Core Space owner.

> 对于之前季度发放的POAP，你可以访问"eSpace Manage"页面的"Set Core Owner"卡片来设置Core Space的所有者。

<img width="1243" alt="image" src="https://github.com/Conflux-Chain/conflux-documentation/assets/17946284/d298d981-b4c0-41fc-8fd9-67e05adcdfa0">

**Comment by darwintree** [comment]
Thank you for contributing! Please leave your (temporary) core space address here for FC airdrop. The airdrop value can be checked [here](https://confluxglobal.notion.site/8e3343b46c214221bf620f46b2f61795?v=111e1e5e40124195b1c8898ef1a56bd9). If anything is missed, please let us know

@intrepidcanadian @jackleeio @jcbm123143 @Last-emo-boy @devBlue4

BTW: apart from the reward, you can claim this quarter's contributor POAP

**Reply by intrepidcanadian** [reply]
Awesome thanks! Core Space Address: cfx:aare6faheu4zgyrth9x1317ddcx19p55gyjfe4rwcx

**Reply by jackleeio** [reply]
Thanks! My address: cfx:aat9mg6m030w598fj2xexvjxkzuv177nza3ruzc61s

**Reply by Last-emo-boy** [reply]
> Thank you for contributing! Please leave your (temporary) core space address here for FC airdrop. The airdrop value can be checked [here](https://confluxglobal.notion.site/8e3343b46c214221bf620f46b2f61795?v=111e1e5e40124195b1c8898ef1a56bd9). If anything is missed, please let us know
> 
> @intrepidcanadian @jackleeio @jcbm123143 @Last-emo-boy @devBlue4
> 
> BTW: apart from the reward, you can claim this quarter's contributor POAP 

Thanks! Here's the address
cfx:aasr3f7xhc7nh8655spe4faku37pdyf9zar6pts9z7

**Reply by darwintree** [reply]
Would know your address for the airdrop @devBlue4 @jcbm123143, we will initiate the airdrop this week. But don't worry your rewards will always be reserved.

**Reply by devBlue4** [reply]
Thanks @darwintree! This is my address: cfx:aanrna48ryk0kamg4xfv384xv7mk9wdfbjurrz0a58
